internal/service/rds/cluster.go

```diff
@@ -237,10 +237,30 @@ func ResourceCluster() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: verify.ValidARN,
 			},
+			"manage_master_user_password": {
+				Type:          schema.TypeBool,
+				Optional:      true,
+				Default:       false,
```

Comment:
This attribute does not need to have a default value specified, as it is optional and will be omitted from the CreateDBCluster() call if not provided by the user. Did you experience a specific test case that was failing without this?